sample_trial/services/init-service/application-dev.yml

```diff
@@ -33,12 +33,23 @@ trial:
       roles:
         - superuser
       userAccount: dc32d94e-1ade-4968-b83e-bab949b7e90c
+    - givenName: qacreate
```

Comment:
I am not sure wee need this user as part of the init service..

Reply:
we've also got a testautomation? I added it to allow flexibility as testautomation and bootstrap user have the same role/permissions. this is a *-dev file, therefore is for dev envs...when we get closer to prod or other near prod envs we would use *-prod or *-preprod etc etc... we could create another profile *-qa but not sure of the benefit